config/production.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+module.exports = {
+  FOO: 'bar'
```

Comment:
I think these should have `PORT`s too otherwise the server won't start in those environments. Or you can move the `PORT` config in development.js to config/index.js so it applies to all environments. Obviously you won't be deploying to staging or production, so this is a nit-pick.